src/App.js

```diff
@@ -1,26 +1,78 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import Home from "./home";
+import Cart from "./cart";
+import { Navbar, Nav, NavItem } from "reactstrap";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+import { productData } from "./data/index";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+export default class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: null,
+      cart: [],
+      cartAddedId: []
+    };
+  }
+
+  addProductToCart = data => {
+    const { cart, cartAddedId } = this.state;
```

Comment:
Please rename this variable `cartAddedId` to `productIds`. It's a bit misleading 😬